esp-hal/src/twai/mod.rs

```diff
@@ -847,13 +859,13 @@ where
                 // Enable /2 baudrate divider
                 T::register_block()
                     .int_ena()
-                    .modify(|r, w| unsafe { w.bits(r.bits() | 0x08) });
+                    .modify(|r, w| unsafe { w.bits(r.bits() | 0x10) });
```

Comment:
🤔 

`/* THIS IS NOT AN INTERRUPT. brp_div will prescale BRP by 2. Only available on ESP32 Revision 2 or later. Reserved otherwise */`

I know we don't care about revisions yet - but maybe have a comment here?

Reply:
👍 We should also add this bit to the PAC, probably. `w.brp_div_not_an_interrupt().set_bit` :D